dashboard/app/models/circuit_playground_discount_code.rb

```diff
@@ -20,11 +20,23 @@ def self.claim(full_discount)
 
     code = nil
     Retryable.retryable(on: ActiveRecord::RecordNotSaved) do
```

Comment:
Does this cause an infinite loop if the `update_all` fails? In particular, does this cause us to repeatedly notify HB? Perhaps a number of retries should be specified?

Reply:
Retryable.retryable [defaults](http://www.rubydoc.info/gems/retryable/2.0.3#Defaults) to 2 tries.

Reply:
Ah, useful to know. LGTM.